views/projects-list/index.html

```diff
@@ -63,11 +67,21 @@ <h2>{{ gettext("yourProjectsLink") }}</h2>
             <div class="project-description">{% if project.description %}{{ project.description | trim }}{% endif %}</div>
           </a>
 
-          <div class="project-ui">
+          <div class="project-ui glitch">
             {% if project.publish_url %}
```

Comment:
the remix and published link shouldn't show up anymore. So your logic might look like:
```django
{% if glitchExportEnabled %}
    {% if project.glitch_migrated %}
    ...
{% else %}
    {% if project.glitch_migrated %}
    ...
{% endif %}
```